frontend/test/metabase/scenarios/native-filters/reproductions/17490.cy.spec.js

```diff
@@ -31,12 +31,10 @@ describe("issue 17490", () => {
      * Cypress fails to click any element that is not "actionable" (for example - when it's covered).
      * In other words, the `.click()` part is essential for this repro to work. Don't remove it.
      */
-    cy.findByPlaceholderText("Find...")
+    cy.findByDisplayValue("Orders");
```

Comment:
actually there is a trailing semicolon

Reply:
yup, just removed it and sent to test again, thanks for the review!